apps/website/src/components/institute/EditPixelForm.tsx

```diff
@@ -0,0 +1,149 @@
+import { type FormEvent, useState } from "react";
+
+import type { MyPixel } from "@/server/trpc/router/donations";
+
+import { formatDateTimeLocal } from "@/utils/datetime";
+
+import { PIXEL_RENAME_LOCK_DURATION_TEXT } from "@/hooks/pixels";
+import { useTimestamp } from "@/hooks/timestamp";
+
+import type { SharedEditPixelProps } from "@/components/institute/EditPixelsForm";
+import MyPixelPreview from "@/components/institute/MyPixelPreview";
+import { PixelIdentifierInput } from "@/components/institute/PixelIdentifierInput";
+import { coordsToGridRef } from "@/components/institute/Pixels";
+import DonationProviderIcon from "@/components/shared/DonationProviderIcon";
+import { Button } from "@/components/shared/form/Button";
+
+const confirmationMessage = (oldIdent: string, newIdent: string) =>
+  `Are you sure you want to change your Pixel label from ${oldIdent} to ${newIdent}? ` +
+  `You will only be able to change your Pixel every ${PIXEL_RENAME_LOCK_DURATION_TEXT}. ` +
+  `The last date to change Pixel labels will be announced on Alveus channels!`;
```

Comment:
```suggestion
const confirmationMessage = (oldIdent: string, newIdent: string) =>
  `Are you sure you want to rename your pixel from "${oldIdent}" to "${newIdent}"? ` +
  `Pixels can only be renamed once every ${PIXEL_RENAME_LOCK_DURATION_TEXT}.`;
```
I think we can reduce the wording here, and I don't think we need to reference the end here.